lib/models/BucketInfo.ts

```diff
@@ -841,4 +851,21 @@ export default class BucketInfo {
         this._objectLockEnabled = enabled;
         return this;
     }
+
+    /**
+     * Get the value of bucket tags
+     * @return - Array of bucket tags
+     */
+    getTags() {
+        return this._tags;
+    }
+    
+    /**
+     * Set bucket tags
+     * @return - bucket info instance
+     */
+    setTags(tags: Array<BucketTag> | []) {
```

Comment:
Same comment as above, I don't think we need or want the `| []`

Reply:
https://github.com/scality/Arsenal/pull/1933/commits/4b19cc2101f4a15666f8746ad8fb12533e9b45d8#diff-4ee3cd5de98073dadb72b27f034f4979480c04388cc4f4538f4895373527f174R867